docs/application/native/guides/multimedia/pose-detection.md

```diff
@@ -0,0 +1,345 @@
+# Pose Detection
+
+Pose Detection is a new feature of Media Vision Inference API since Tizen 6.0. This feature provides landmark detection. Besides, it defines landmarks and parts of a human body to help detect a human pose with Motion Capture (MoCap) file, which you can create or edit using various tools.
+
+## Background
+
+In Tizen, human body pose landmarks and body parts are defined as follows:
+
+**Figure: Definition of human body pose landmarks and body parts**
+
+![Body pose](./media/mediavision_pose_tizen_def.png)
+
+The Pose landmark detection models are available in Open Model Zoo such as [hosted model zoo](https://www.tensorflow.org/lite/guide/hosted_models#floating_point_models) or public GitHub site such as [public pose model](https://github.com/tyoungroy/PoseEstimationForMobile). The public pose models provide landmark information, such as the number of landmarks and locations. To use them correctly, you must map the information to landmarks based on the definition. For example, you can use the [public pose model](https://github.com/tyoungroy/PoseEstimationForMobile), which provides 14 landmarks as follows:
```

Comment:
With [previous comment](https://github.com/Samsung/tizen-docs/pull/1244#discussion_r491663987), I accepted the link to @tyoungroy's personal repository as kind of official repository for the pose model. This comment is just a question. Is there any reason that you didn't link to the original repository that you forked from?